### DISCUSSION TITLE
stat-computing-r-python/part-tools/03-scripts-notebooks

### DISCUSSION THREAD

**Discussion by giscus** [srvanderplas/stat-computing-r-python#5] [General]
# stat-computing-r-python/part-tools/03-scripts-notebooks

https://srvanderplas.github.io/stat-computing-r-python/part-tools/03-scripts-notebooks.html

**Comment by extrastats** [comment]
I am struggling with the script part in the second "your turn" section. 
I have downloaded scripts.zip, unzipped it, and have it in my statistics folder. 
When opening it up with terminal in the manner you mentioned, and putting in the code given, I am getting this message: "Fatal error: cannot open file 'words.R': No such file or directory"

**Comment by srvanderplas** [comment]
Figured out why this is happening - 

The terminal in RStudio does not update with the working directory you're in. So you have to explicitly change directory IN THE TERMINAL to make it work. I am adding a short guide to terminal commands and will explicitly note that you have to do this in the example text.

**Reply by srvanderplas** [reply]
I have added some [additional information in the "other topics" section of the textbook about shell commands that may help you diagnose the problem in more detail.](https://srvanderplas.github.io/stat-computing-r-python/graveyard.html#sec-shell-commands)
